mods/boats/init.lua

```diff
@@ -224,10 +224,10 @@ minetest.register_craftitem("boats:boat", {
 
 	on_place = function(itemstack, placer, pointed_thing)
```

Comment:
Your suggestion to not pass placer: `function(itemstack, pointed_thing)`
Would just lead an ObjectRef being passed as `pointed_thing`. Since the ObjectRef doesn't have above/under, it would crash.
So you **have** to mention it, even if you don't use it.